.github/workflows/dockerhub-humble.yml

```diff
@@ -1,6 +1,6 @@
-# This workflow will build and push official docker images for Focal OS when new push occurs 
```

Comment:
because of file name change and changes being pretty similar, I am not able to make them compare with correct files. But it should be easy to see the changes. 
focal becomes noetic. jammy becomes humble